src/crypto/crypto_util.cc

```diff
@@ -158,6 +157,8 @@ void InitCryptoOnce() {
         break;
     }
   }
+#endif
```

Comment:
Could you add `// OPENSSL_IS_BORINGSSL` to the `#endif` _below_ this one? Multiple consecutive `#endif`s (due to nested `#if`s) tend to be confusing without indentation.